alertMaintenance.js

```diff
@@ -76,22 +76,19 @@ const updateAlert = async function(info, pgClient, discordClient, isComplete){
 		}
 	}
 
-
-	pgClient.query("SELECT messageID, channelID FROM alertMaintenance WHERE alertID = $1;", [info.instanceId])
```

Comment:
If I understand this correctly, these two lines should execute the same way because the `await` suspends execution of the current function body and allows other events to be processed.